pds4indextools/pds4_create_xml_index.py

```diff
@@ -198,271 +701,781 @@ def update_nillable_elements_from_xsd_file(xsd_file, nillable_elements_info):
+                                            'like *, ?, and **. If supplying multiple '
+                                            'patterns, separate with spaces. Surround '
+                                            'each pattern with quotes.')
+
+    index_file_generation.add_argument('--output-index-file', type=str,
+                                       help='Specify the location and filename of the '
+                                            'index file.')
+
+    index_file_generation.add_argument(
+        '--add-extra-file-info',
+        type=lambda x: validate_comma_separated_list(x, valid_add_extra_file_info),
+        help='Add additional columns to the final index file. If specifying multiple '
+        'column names, supply them as separate arguments separated by spaces.')
+
+    index_file_generation.add_argument(
+        '--sort-by',
+        type=str,
+        help='Sort resulting index file by one or more columns. Must be specified by '
+             'either the full XPath header or the simplified version if using '
+             '--simplify-xpaths. To see all available options, use '
+             '--output-headers-file.')
+
+    index_file_generation.add_argument('--fixed-width', action='store_true',
+                                       help='Create an index file with fixed-width '
+                                            'columns.')
+
+    index_file_generation.add_argument('--clean-header-field-names', action='store_true',
+                                       help='Rename column headers such that they only '
+                                            'contain characters permissible in variable '
+                                            'names.')
+
+    index_file_generation.add_argument(
+        '--simplify-xpaths',
+        action='store_true',
+        help='If specified, only writes the tags of unique XPaths to output files. Any '
+             'values with duplicate values will still use their full XPath.')
+
+    limiting_results = parser.add_argument_group('Limiting Results')
+    limiting_results.add_argument('--limit-xpaths-file', type=str,
+                                  help='Optional text file specifying which elements to '
+                                       'scrape. If not specified, all elements found in '
+                                       'the label files are included.')
+
+    limiting_results.add_argument('--output-headers-file', type=str,
+                                  help='Specify the location and filename of the XPath '
+                                       'headers file.')
+
+    label_generation = parser.add_argument_group('Label Generation')
+    label_generation.add_argument('--generate-label', type=str, nargs=1,
+                                  help='Generate a PDS4 label for the generated index '
+                                       'file called "<output_index_file>_label.xml". '
```

Comment:
Why is there a `_label` on the filename? Normally the label for a file is the same name as the data file, just with an `.xml` extension.

Reply:
Easier to distinguish on first glance. I'll get rid of it.